backend/services/user/customer.js

```diff
@@ -29,11 +29,14 @@ const customerCreate = async (request, response) => {
 const customerLogin = async (request, response) => {
   try {
     const { email, password } = request.body;
-    const user = (await query(
-      'SELECT customer_id, order_count, points, name FROM customers where exists( select 1 from users u where email = $1 and password = $2 and customer_id = u.user_id)',
+    let user = (await query(
+      'SELECT customer_id , order_count, points, name FROM customers where exists( select 1 from users u where email = $1 and password = $2 and customer_id = u.user_id)',
       [email, password],
     )).rows[0];
     // append info to user object
+    user = { customerId: user.customer_id, order_count: user.order_count, ...user };
```

Comment:
Instead of using the same user object, maybe create a new object entirely. This will remove the need to delete attributes as well.